umiCollapse.wdl

```diff
@@ -160,15 +160,15 @@ workflow umiCollapse {
     if (doBamQC) {
         call bamQC.bamQC as preDedupBamQC {
             input:
-                bamFile = mergeLibrary.mergedBam,
+                bamFile = mergeLibrary.mergedBam ,
```

Comment:
I have a feeling that I have already seen this extra space before. Deja vu, Matrix is glitching

Reply:
resolved